README.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+This is a demo module for Islandora CLAW. It illustrates the modeling of a content type and creation of a drupal module for CLAW.
+
+
+## Steps to create claw_porcus
+### 1 - Create a new Media bundle
+* Go to `/admin/structure/media/add`
+* Provide a label to the bundle: PORCUS_TEXT
+* Leave the type provider as Generic media
+* Go to the Manage fields of the new media bundle: `/admin/structure/media/manage/porcus_text/fields`
+* Add a new file field
```

Comment:
Should mention that the file field is already defined in `islandora` core.  Something like "Add the existing `field_file` field to the bundle."